setup/generate_usptream_rust_keylime_code_coverage/test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+# for Packit PRs we would be uploading code coverage files unless forbidden
+[ -n "${PACKIT_SOURCE_URL}" -a -z "${UPLOAD_COVERAGE}" ] && UPLOAD_COVERAGE=1
+UPLOAD_URL=https://transfer.sh
+
+
+rlJournalStart
+
+    rlPhaseStartSetup "Collect code coverage for rust components"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        #delete coverage script, in dir are mandatory only coverage files
+        rlRun "pushd ${__INTERNAL_limeCoverageDir}"
+        # export in format for user
+        rlRun "grcov . --binary-path /usr/local/bin/keylime_agent -s . -t html --branch --ignore-not-existing -o e2e_coverage.html"
+        # export coverage report in format compatible for codecov
+        rlRun "grcov . --binary-path /usr/local/bin/keylime_agent -s . -t lcov --branch --ignore-not-existing -o e2e_testsuite.txt"
+        # create tar file for uploading coverage file
+        rlRun "tar --create --file e2e_testsuite.tar e2e_coverage.html"
+        rlFileSubmit  e2e_testsuite.tar
+        rlFileSubmit e2e_testsuite.txt
+        if [ "${UPLOAD_COVERAGE}" == "1" ]; then
+            # upload coverage.tar.gz
+            rlRun -s "curl --upload-file e2e_testsuite.tar $UPLOAD_URL"
```

Comment:
would it make sense to upload individual files only when they really exist? `if [ -f FILE ]; then ...`